### ISSUE TITLE
Failed to detect the version of linux

### ISSUE THREAD

**Issue by drc0nc** [securestate/king-phisher#206]
## Issue Description
Failed to detect the version of linux on new install. I'm running the newest ubuntu server 17.04 

## Reproduction Steps
Upon fresh install the script gives the failed to detect version of linux.

## Environment Details

Host OS: Ubuntu 17.04
King Phisher Version: newest pull

## Error Details / Stack Trace

```
If you are opening an issue regarding the client, copy text from the "Error
Details" of the error dialog in the client here. If you are opening an issue
regarding the server copy the relevant lines from the logs here.
```

**Comment by zeroSteiner** [comment]
@wolfthefallen looks like the install script needs to be updated to identify 17.04. This'll be an easy fix.

**Comment by drc0nc** [comment]
I forgot to mention that I'm currently in the dev branch

**Comment by zeroSteiner** [comment]
Fixed in commit c64bbd80b4af54ffb7a696f7ea2bd817350fdf65 which is available in both the `master` and `dev` branches.  Thanks for beta testing!
